package.json

```diff
@@ -34,7 +34,7 @@
     "react-dom": "18.2.0",
     "react-hook-form": "^7.51.3",
     "react-hot-toast": "^2.4.1",
-    "tailwind-merge": "^2.2.2",
+    "tailwind-merge": "^2.5.3",
```

Comment:
Why was this updated?

Reply:
When I pulled the most recent changes from the develop branch those dependencies were updated .

Reply:
@kepsteen - the develop branch currently shows this dependency as `2.2.2`.